adrs/6860374633/README.md

```diff
@@ -0,0 +1,126 @@
+# Error CloudEvents Adapter
+
+- **State:** Reviewing
+- **Created:** 2020-11-10
+- **Tags:** nodejs
+
+This document describes how to Errors events into a CloudEvents.
+
+## Table of Contents
+
+- [Error CloudEvents Adapter](#error-cloudevents-adapter)
+  - [Table of Contents](#table-of-contents)
+  - [Purpose](#purpose)
+  - [Context Attributes Extension](#context-attributes-extension)
+    - [dataschema](#dataschema)
+    - [data](#data)
+  - [Event Data](#event-data)
+    - [errormessage](#errormessage)
+    - [errorlink](#errorlink)
+    - [metadata](#metadata)
+  - [Example](#example)
+
+## Purpose
+
+Errors are everywhere. However, error's producers tend to describe errors
+differently.
+
+The lack of a common way of describing errors means developers are continually
+re-learning how to consume errors. That also limits the potential for libraries,
+tooling, and infrastructure to aid of error data across delivery environments,
+like SDK or logger systems. The portability and productivity that can be
+achieved from error data are hindered overall.
+
+This ADR is a specification for describing error data in standard formats to
+provide interoperability across services, platforms, and systems.
+
+## Context Attributes Extension
+
+### dataschema
+
+- Constraints:
+  - REQUIRED
+  - MUST be <https://github.com/straw-hat-team/adr/tree/master/adrs/6860374633/v1/dataschema.json>
```

Comment:
This is a broken link

Reply:
yeah it is until we merge the ADR, I am not sure how to solve the issue since we need an URL to it.